Examples/JamitFoundationExample/App/Sources/ListView/ListItem/ListItemViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import JamitFoundation
+
+struct ListItemViewModel: ViewModelProtocol {
+    let title: String?
+
+    init(title: String? = ListItemViewModel.default.title) {
+        self.title = title
+    }
+}
+
+extension ListItemViewModel {
+    static let `default`: ListItemViewModel = .init(
```

Comment:
You can reduce the duplication of struct name by `Self`.

Reply:
Done.